mail_sendgrid/models/mail_mail.py

```diff
@@ -187,15 +187,15 @@ def _prepare_sendgrid_data(self):
         subject = self.subject and self.subject.encode(
             "utf_8") or "(No subject)"
         personalization.subject = subject
-        addresses = list()
```

Comment:
If you remove the variable, line 197 will fail in case there is no email_to defined, because the list won't be initialized.